src/flow-hash.c

```diff
@@ -506,9 +506,7 @@ static inline Flow *FlowSpareSync(ThreadVars *tv, FlowLookupStruct *fls,
         f = FlowQueuePrivateGetFromTop(&fls->spare_queue);
         spare_sync = true;
     }
-#ifdef UNITTESTS
```

Comment:
why not support the counters from this path as well? Seems useful to me.

Reply:
Currently the only user of flow bypass manager is af-packet and with the new implementation (5.1) the thread starts run the init function and leave as the recurring check function is not implemented. The currently used init function read the pinned maps and create the flows based on the data collected. So we could have a counter but it will be for a one shot function at start.

Reply:
I don't understand this, what is version 5.1 here?

So when does this thing run? Does it even make sense to use a spare pool here?

Reply:
Oups, please read Suricata 5.0.x not 5.1.

It runs at start of Suricata to load the flows from the flow bypass table stored in kernel.This is only active when the eBPF maps are pinned to resist a Suricata restart.

Reply:
If its not performance critical lets keep things as simple as possible.

Reply:
OK, in this case I think we have no active code to add here. I will keep the doc update for later to not add more on top of your current work.